tashiad/fitlit
# Feature design

## What's this PR do?

This PR adds structure and styling for the dashboard. It includes all HTML necessary to build the layout and all CSS rules to make the dashboard look user-friendly.

## Where should the reviewer start?

The reviewer should start with index.html to get the basic structure, then look at scripts.js to see what HTML is being added through the DOM and then look at styles.css to see how the styling is done.

## How should this be manually tested?

To see how the page looks, the reviewer should run index.html through their browser.

## Any background context you want to provide?

All data appearing should be coming from the large datasets provided.

## What are the relevant tickets?

Any issues with tags of dashboard.

## Questions:

How do we get moment to work?
